### PR TITLE
Task/language confirmation styling

### LINKED ISSUE #795: Styling for Language Requirement Confirmation Checkbox
Follow up to issue #715

- Checkbox for language requirement confirmation has no required red asterisks.
- Checkbox for language requirement has no helper text when validation fails.

### PR DESCRIPTION
**Pull request by yonikid15**
Resolves #795 

### Notes:
- Added in new styling for language confirmation box, in the step 1 page form for the job applying process. 
checkbox1
checkbox2

### CODE REVIEW

File: resources/assets/js/app.js [hunk 1]
```diff
@@ -286,6 +286,18 @@
             "<span class='form__required'><i class='fa fa-asterisk' aria-label='Asterisk'></i></span>"
           );
       });
+
+      // Required field for checkbox input
+      $(`input[type="checkbox"]:required`).each(function(e) {
```
Comment (Khristophor) [inline resources/assets/js/app.js]: I spy a template literal ;)

File: resources/assets/js/app.js [hunk 2]
```diff
@@ -393,6 +405,25 @@
         // unfocus item to trigger style
         $(this).blur();
       });
+
+      $(`#language_requirement_confirmed`).change(function(e) {
```
Comment (Khristophor) [inline resources/assets/js/app.js]: One here too

**Comment by Khristophor** [comment]
Nice work refactoring the language requirement twig template 👍

File: resources/assets/js/app.js (hunk 1 above)
Comment (yonikid15) [reply resources/assets/js/app.js]: damn haha

**Pull request merged**